internal/core/src/segcore/FieldIndexing.cpp

```diff
@@ -17,9 +17,13 @@
 #include "common/SystemProperty.h"
 #include "segcore/FieldIndexing.h"
 #include "index/VectorMemNMIndex.h"
+#include "IndexConfigGenerator.h"
 
 namespace milvus::segcore {
 
+VectorFieldIndexing::VectorFieldIndexing(const FieldMeta& field_meta, const FieldIndexMeta& field_index_meta, int64_t segment_max_row_count, const SegcoreConfig& segcore_config): FieldIndexing(field_meta, segcore_config), config_(std::make_unique<VecIndexConfig>(segment_max_row_count, field_index_meta, segcore_config)), sync_with_index(false) {
```

Comment:
The code should be formatted, as it obviously exceeded the length limit in .clang-format